build/sdk/js/templates/index.mustache

```diff
@@ -72,5 +72,11 @@
     'User-Agent': 'public-api-sdk-generator/{{projectVersion}}'
   };
 
+  exports.setSandbox = function(isSandbox) {
+    exports.ApiClient.instance.basePath = isSandbox ?
+      'https://api-sandbox.shutterstock.com' :
+      'https://api.shutterstock.com';
```

Comment:
Would rather only have the overload if isSandbox, otherwise default to the openapi basepath.

Reply:
don't have access to {{basePath}} here unfortunately, which is why I didn't do that...